lib/model/migrations/20250415-01-client-audit-remainder.js

```diff
@@ -0,0 +1,34 @@
+// Copyright 2025 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+const up = async (db) => {
+  await db.raw(`
+    ALTER TABLE client_audits
+      ADD COLUMN "user" TEXT,
+      ADD COLUMN "change-reason" TEXT
+  `);
+
+  await db.raw(`
+    UPDATE client_audits
+    SET 
+      "user" = remainder->>'user',
+      "change-reason" = remainder->>'change-reason'
+    WHERE remainder IS NOT NULL;
+  `);
```

Comment:
Once we move `user` and `change-reason` into their own columns, how about removing them from `remainder`? We know the `client_audits` table can get very large, so it might help to keep `remainder` as small as possible.

Reply:
We could do this, but i'm weirdly concerned about the down migration and putting the user/change-reason column data back into remainder? I could just not have that in the down migration since it will probably never be run in a real environment?

Reply:
Maybe it wouldn't be hard to add that to the down migration? We could update `remainder` again as part of the down migration. But I agree that it's not essential. We often don't write a down migration at all. If someone is having to run a down migration in production, something has already gone wrong.

Reply:
Added to the down migration and tested locally!